stock_by_warehouse/static/src/js/widget.js

```diff
@@ -42,5 +43,25 @@ var ShowPaymentLineWidget = form_common.extend({
     },
 });
 
+
+var ShowProductAvailability = form_common.extend({
+    _render: function(){
+        var self = this;
+        this._super.apply(this, arguments);
+        this.info = JSON.parse(this.value);
+
+        this.$el.html(QWeb.render('StockAvailabilityAlert', {
+            'locations_available':  formats.format["integer"](
+                self.info.available_locations || 0, this.field, {
+                    digits: self.info.digits || 0}),
+        }));
```

Comment:
can you modify the original polease... to manage all toghether.

Reply:
Done :smile: